salt/cloud/clouds/azurearm.py

```diff
@@ -8,7 +8,7 @@
 The Azure cloud module is used to control access to Microsoft Azure
 
 :depends:
-    * `Microsoft Azure SDK for Python <https://pypi.python.org/pypi/azure>`_ == 2.0rc2
+    * `Microsoft Azure SDK for Python <https://pypi.python.org/pypi/azure>`_ >= 2.0rc3
     * `Microsoft Azure Storage SDK for Python <https://pypi.python.org/pypi/azure-storage>`_
```

Comment:
Storage is splitted into two parts:
- azure-storage (data): There is a huge number of breaking changes from 0.30 to 0.31. I recommend 0.32. I don't what you use currently
- azure-mgmt-storage (mgmt): There is some small breaking changes in 2.0.0RC5 compared to 2.0.0RCx before.  Should be the last breaking changes before the stable, since we updated to the latest REST API Version of Storage Management. See ChangeLog for details:
https://github.com/Azure/azure-sdk-for-python/releases/tag/v2.0.0rc5